src/pages/en/tutorial/0-introduction/1.md

```diff
@@ -0,0 +1,91 @@
+  - set up the necessary online accounts to deploy your site to the web
+</Goals>
+
+This tutorial is designed to be accessible to new web developers, as well as devs who are simply new to Astro. It assumes some basic familiarity with **HTML**, **Markdown**, **CSS**, and a little **JavaScript**. But, you will be able to learn a lot of what you need to know during the tutorial itself!
+
+:::tip
+You can also complete this tutorial using a web browser **instead of installing any local software**! You can access both the command line and a code editor together in a cloud development environment like [StackBlitz](https://stackblitz.com) or [CodeSandbox](https://codesandbox.io).
+
+Visit [astro.new](https://astro.new) and click to open the "Completely Empty" project in the editor of your choice.  "Fork" the project to save it to your own account. Then, skip ahead to setting up [Source Control](#source-control) for your tutorial project.
+:::
+
+## Development Tools Needed
+
+### Terminal
+
+You will use the **command line (terminal)** to create your Astro project and to run key commands to build, develop, and test your site.
+
+You can access the command line through a local terminal program for your operating system. Common applications include **Terminal** (MacOS/Linux), **Command Prompt** (Windows), and **Termux** (Android). One of these will probably already be on your machine. 
+
+You will also need to have **Node.js** installed. To check to see whether you already have a compatible version (v.14.18.0, or v16.12.0 or higher) installed: 
+
+1. Launch your terminal and type the following into the command line:
+
+    ```sh
+    node -v
+    ```
+
+2. If necessary, follow the instructions to [install Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) via the command line.
+
+### Code Editor
+
+Additionally, you will need to download and install a **code editor** to write your code. We will use **VS Code** in this tutorial, but you can use any editor for your operating system.
+
+1. [Download and install VS Code](https://code.visualstudio.com/#alt-downloads) or another code editor of your choice. 
+
+
+### Source Control
+
+Many software projects use a version control system to track changes in a project over time. In this tutorial, you will use a free **online git provider** and will not need to know how to use git commands in the terminal. All git actions in this tutorial will be performed through menu items in your code editor and will sync your **repository** (files and version history) to the cloud. This tutorial will use **GitHub** for online source control, but you are welcome to use your preferred online git provider.
+
+1. Create a free account at [GitHub](https://github.com) (or your preferred cloud git provider).
+
+    Make a note of your username. You will view your account, and any projects you create at `https://github.com/username`
+
+### Continuous Build and Deployment
```

Comment:
I could totally see leaving deployment until the end, I imagine this was a very deliberate choice!

Reply:
Very deliberate!  :smile: 

We've seen a couple of streamers immediately deploy, before doing anything to their created project. It's a good way to check that everything is working before you've done anything that can mess it up. If it doesn't deploy NOW, you've got bigger problems... and they're probably not your problems!

Plus, I do think it goes towards quick wins - "having a website immediately" and it's on the web! Already!